File: 백준/Bronze/10818. 최소， 최대/최소， 최대.cs
using System;

class Program {
    static void Main(string[] args) {
        int N = int.Parse(Console.ReadLine());
        string[] str = Console.ReadLine().Split();
        int min = int.Parse(str[0]), max = int.Parse(str[0]);
        for (int i = 0; i < N; ++i) {
            if (int.Parse(str[i]) < min)
                min = int.Parse(str[i]);
            if (int.Parse(str[i]) > max)
                max = int.Parse(str[i]);
        }
        Console.WriteLine("{0} {1}", min, max);
    }
}

File: 백준/Bronze/10818. 최소， 최대/최소， 최대.js
const input = require('fs').readFileSync('/dev/stdin').toString().split('\n');
const N = parseInt(input[0]);
const nums = input[1].split(' ').map(Number);

let min = nums[0], max = nums[0];

for (let i = 1; i < N; ++i) {
    if (nums[i] < min) min = nums[i];
    if (nums[i] > max) max = nums[i];
}

console.log(min, max);